client.cpp

```diff
@@ -490,3 +491,17 @@ PHP_METHOD(VoltClient, drain)
         RETURN_FALSE;
     }
 }
+
+PHP_METHOD(VoltClient, close)
+{
+    zval *zobj = getThis();
+    voltclient_object *obj = (voltclient_object *)zend_object_store_get_object(zobj TSRMLS_CC);
+
+    try {
+        voltdb::ConnectionPool::pool()->closeClientConnection(*(obj->client));
```

Comment:
There is a memory leak here. You should delete `obj->client` after closing it, and then set it to NULL.